crates/fuel-core/src/service/adapters.rs

```diff
@@ -85,6 +133,16 @@ pub struct BlockImporterAdapter {
         Arc<fuel_core_importer::Importer<Database, ExecutorAdapter, VerifierAdapter>>,
 }
 
+impl BlockImporterAdapter {
+    pub fn events(&self) -> BoxStream<SharedImportResult> {
+        use futures::StreamExt;
```

Comment:
Do you have a reason why you'd put the import here rather than at the top. I see this sometimes and I'm curious what the justification for it is. I don't have anything against it.

Reply:
It is used only in one place in that file. If you decide to remove that code later, you don't need to clean up imports too=) Plus it creates fewer conflicts with other people.